rustfmt-core/rustfmt-lib/tests/source/issue-4211.rs

```diff
@@ -0,0 +1,13 @@
+trait SomeTrait<T>
```

Comment:
In a case like this that's an idempotence check the file under `source` can be omitted.